runtime/src/iree/base/internal/BUILD.bazel

```
@@ -101,6 +101,38 @@ iree_runtime_cc_test(
     ],
 )
 
+iree_runtime_cc_library(
+    name = "bitmap",
+    srcs = ["bitmap.c"],
+    hdrs = ["bitmap.h"],
+    deps = [
+        "//runtime/src/iree/base",
+    ],
```

missing dep (see my other comment)
```suggestion
    deps = [
        ":internal",    
        "//runtime/src/iree/base",
    ],
```